.github/workflows/pr-unit-tests.yml

```diff
@@ -15,7 +15,7 @@ permissions: {}
 jobs:
     test:
         if: ${{ github.event.pull_request.user.login != 'github-actions[bot]' }}
-        name: PHP ${{ matrix.php }} WP ${{ matrix.wp }} ${{ matrix.hpos && 'HPOS' || '' }}
+        name: PHP ${{ matrix.php }} WP ${{ matrix.wp }} ${{ matrix.hpos && 'HPOS' || '' }} - ${{ matrix.unittests }}
```

Comment:
If HPOS is disabled, this would lead to two spaces being in the name. As the "required" checks need to be configured based on this name this could become pretty confusing. Maybe it's better to move the optional "HPOS" to the end of the string?

Reply:
Good point! Will fix.